### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by yuansutina**
added icon

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,11 +4,226 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (Codyzhao) [inline src/index.html]: This is repeated from line 5

File: src/index.html [hunk 2]
```diff
@@ -4,11 +4,226 @@
...
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+<style>
```
Comment (Codyzhao) [inline src/index.html]: Best to put these styles into `./css/index.scss`
Also please make sure you add some formating for these styles.
https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics#Anatomy_of_a_CSS_ruleset

File: src/index.html [hunk 3]
```diff
@@ -4,11 +4,226 @@
...
+ right: 0;
+ bottom: 0;
+}
+</style>
   </head>
   <body>
-    <div id="main"></div>
+    <div class="main">
+    
     <script src="./js/index.js"></script>
```
Comment (Codyzhao) [inline src/index.html]: Javascript should be included at the end of the document.
The reason is JS should be executed after the HTML DOM is loaded.

I think your point is you might use the  document.ready from jQuery, but I think we might not require jQuery for this homework.

File: src/index.html [hunk 4]
```diff
@@ -4,11 +4,226 @@
...
+    <h1>John Roberts</h1>
+    <h2>Home</h2>
+    <button style="margin-left: 1050px">DOWNLOAD MY FULL RESUME</button>
+    <img SRC="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_248,h_244,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.webp"  class="img-circle" alt="Jhon">
```
Comment (Codyzhao) [inline src/index.html]: lowercase on attributes plz: `SRC`
Only one space between each attribute.

File: src/index.html [hunk 5]
```diff
@@ -4,11 +4,226 @@
...
+        <ol class="carousel-indicators">
+          <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+          <li data-target="#myCarousel" data-slide-to="1"></li>
+          <li data-target="#myCarousel" data-slide-to="2"></li>
+          <li data-target="#myCarousel" data-slide-to="3"></li>
+        </ol>
+        <!-- Wrapper for slides -->
+      <div id="divcss5" class="carousel-inner">
+      <div class="item active">
+      <h2>ABOUT ME</h2>
```
Comment (Codyzhao) [inline src/index.html]: Indentation

File: src/index.html [hunk 6]
```diff
@@ -4,11 +4,226 @@
...
   <body>
-    <div id="main"></div>
+    <div class="main">
+    
     <script src="./js/index.js"></script>
+    <h1>John Roberts</h1>
+    <h2>Home</h2>
+    <button style="margin-left: 1050px">DOWNLOAD MY FULL RESUME</button>
+    <img SRC="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_248,h_244,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.webp"  class="img-circle" alt="Jhon">
+    <p style="font-size:20pt;color:DodgerBlue;text-align:center">BUSINESS EXPERT<p>
```
Comment (Codyzhao) [inline src/index.html]: Might consider to move the styles at least to the header styles section. 
Inline style is not recommended due to hard to override later on.

File: src/index.html [hunk 7]
```diff
@@ -4,11 +4,226 @@
...
+      <div class="item active">
+      <h2>ABOUT ME</h2>
+      <div class="row">
+      <div class="col"><img SRC="https://static.wixstatic.com/media/aa0402eb9ba2430d9d0620b59556efca.png/v1/fill/w_32,h_32,al_c,q_80,usm_0.66_1.00_0.01/aa0402eb9ba2430d9d0620b59556efca.webp"  class="img-circle" alt="linkedin"></div>
+      <div class="col"><img SRC="https://static.wixstatic.com/media/4057345bcf57474b96976284050c00df.png/v1/fill/w_32,h_32,al_c,q_80,usm_0.66_1.00_0.01/4057345bcf57474b96976284050c00df.webp"  class="img-circle" alt="facebook"></div>
+      <div class="col"><img SRC="https://static.wixstatic.com/media/870f97661ed14a5bb2d96ecbddec0aed.png/v1/fill/w_32,h_32,al_c,q_80,usm_0.66_1.00_0.01/870f97661ed14a5bb2d96ecbddec0aed.webp"  class="img-circle" alt="twitter"></div>
+      </div>
+     
+     <div class="item">
+     <table>
```
Comment (Codyzhao) [inline src/index.html]: indentations need to be fixed.

File: src/index.html [hunk 8]
```diff
@@ -4,11 +4,226 @@
...
+        <h3>L.A. University of Applied Sciences</h3>
+        <h3>2002 - 2006</h3>
+        <p>This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.</p>
+        </div>
+        </div>
+      </div>
+      <div class="item">
+        <h2>HOBBIES & INTERESTS</h2>
+        <div class="row">
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSf71rBnViRBrals8RJieIrnGbQj8kNO8X-8cbRNKKDDnaQauOH" /></div>
```
Comment (Codyzhao) [inline src/index.html]: missing `alt` attribute in the image tag

File: src/index.html [hunk 9]
```diff
@@ -4,11 +4,226 @@
...
+     </tr>
+     <tr>
+     <td>Address</td>
+     <td>500 Terry Francois Street San Francisco, CA 94158</td>
+     </tr>
+     </table>
+     <h2>CONTACT ME</h2>
+     <form action="" method="post" >
+     <label for="user">Name:</lable>
+     <input type="text" id=user name="user" value="" />
```
Comment (Codyzhao) [inline src/index.html]: `id="user"`

File: src/index.html [hunk 10]
```diff
@@ -4,11 +4,226 @@
...
+     <label for="subjuct">Subjuct:</lable>
+     <input type="text" id=subjuct name="subjuct" value="" />
+     <br>
+     <label for="message">Massage:</lable>
+     <textarea name="textarea" value="" rows="6" cols="25" ></textarea>
+     <br>
+     <input type="button" value="Submit">
+     </form>
+
+     <button >DOWNLOAD MY FULL RESUME</button>
```
Comment (Codyzhao) [inline src/index.html]: `<button>`

File: src/index.html (hunk 7 above)
Comment (hustKiwi) [reply src/index.html]: Hi @yuansutina , have you follow the instruction to set up the node modules for your local development? The reason why I am asking is that if you installed node packages correctly, it should automatically format your code when your committing.

Please double-check the [instruction wiki](https://github.com/nzittraining/cv-assignment/wiki/Assignment-Requirement) and re-install your node packages. Thanks

Comment (yuansutina) [reply src/index.html]: That is way my indentation have problem. I try to install couple times all is done, But when I running it does not work. So I used note finished my work.

File: src/index.html (hunk 6 above)
Comment (yuansutina) [reply src/index.html]: Move to the index.scss, but for other problem is if I used <p></p> other text, they will all follow this style. How to write only for specifying text？ Thank you.

File: src/index.html (hunk 8 above)
Comment (yuansutina) [reply src/index.html]: already added thanks

File: src/index.html (hunk 9 above)
Comment (yuansutina) [reply src/index.html]: <label for="name">Name:</lable>
     <input type="text" id=name name="name" value="" />

Comment (yuansutina) [reply src/index.html]: <label for="name">Name:</label>
     <input type="text" id=name name="name" value="" />

File: src/index.html (hunk 6 above)
Comment (Codyzhao) [reply src/index.html]: Styling is not within the scope of this homework.

Nick will be talking about CSS selectors in the next lessons. So don't worry too much about the CSS part of things for now.

Comment (yuansutina) [reply src/index.html]: OK， thank you.

File: src/index.html (hunk 10 above)
Comment (yuansutina) [reply src/index.html]: What is means <button>?

Comment (Codyzhao) [reply src/index.html]: plz remove the extra empty space :P

Comment (Codyzhao) [reply src/index.html]: It seems that no-biggie, but a good programmer always care about tiny details :)

Comment (yuansutina) [reply src/index.html]: haha, OK
